### ISSUE TITLE
Getting "app_id[Wits] download failed[16]" during install?

### ISSUE THREAD

**Issue by jfrux** [Samsung/Wits#8]
```
Start Wits............
? Input your Application Path : www
? Input your Application width (1920 or 1280) : 1920
? Input your TV Ip address(If using Emulator, input 0.0.0.0) : 10.8.82.242
? Input your port number : 8498
? Do you want to launch with chrome DevTools? :  No
{ baseAppPath: 'www',
  width: '1920',
  ip: '10.8.82.242',
  port: '8498',
  isDebugMode: false }
connect to....10.8.82.242
10.8.82.242:26101 is already connected
Success to connect 10.8.82.242
List of devices attached
10.8.82.242:26101       device          UN43KU6300
emulator-26101          device          T-samsung-5.0-x86
? Select the device to launch Wits : 10.8.82.242:26101
secure_protocol:enabled
intershell_support:disabled
filesync_support:push
usbproto_support:enabled
sockproto_support:enabled
syncwinsz_support:enabled
rootonoff_support:disabled
zone_support:disabled
multiuser_support:disabled
cpu_arch:armv7
profile_name:tv
vendor_name:Samsung
can_launch:tv-samsung-public_-_tv-samsung-partner
platform_version:2.4.0
product_version:2.0
sdbd_version:2.2.31
sdbd_plugin_version:1.0.0
sdbd_cap_version:1.0

appInstallPath
content src is : index.html

Start packaging Samsung Tizen TV Platform......
Tizen CLI 2.4.48
Setting configuration is succeeded....

Build Web Application:  100% (10/10)

BUILD SUCCESSFUL

Output path : <redacted>/tizen/.buildResult
Excluded : [.build/*, .build, .sign/*, .sign, webUnitTest/*, webUnitTest, .externalToolBuilders/*, .externalToolBuilders, .buildResult/*, .buildResult, .settings/*, .settings, .package/*, .package, .tproject, .project, .sdk_delta.info, .rds_delta, *.wgt, .tizen-ui-builder-tool.xml]
Total time: 00:00:00.480
Author certficate: <redacted>/release-tizon.p12
Distributor1 certificate : <redacted>/tizen-distributor-signer.p12
Excludes File Pattern: {.manifest.tmp, .delta.lst}
Ignore File: <redacted>/tizen/.buildResult/.manifest.tmp
Package File Location: <redacted>/tizen/.buildResult/Wits.wgt
Package created at <redacted>/tizen/build/Wits.wgt
uninstall Wits
app_id[Wits] uninstall start
app_id[Wits] uninstall failed[132]
spend time for wascmd is [35]ms
cmd_ret:0

[warning] Failed to uninstall Wits
error: too few arguments
info: sdb push <local> <remote> [--with-utf8]
exec: error: too few arguments

install Wits
package_path Wits.wgt
was_install_app return WAS_TRUE
app_id[Wits] install start
app_id[Wits] download failed[16]
spend time for wascmd is [32]ms
cmd_ret:0

Failed to install Wits
✨  Done in 11.64s.
```

**Comment by pwsses** [comment]
Hi 
Thank you for report issue.
Unfortunately, Wits support Platform after 2017 Samsung Smart TV.
I think, your device is 2016 TV device. (UN43KU6300)
I wrote support platform spec in Samsung Develop site (https://developer.samsung.com/tv/develop/extension-libraries/wits/)
but I was missing gitHub. apologize sincerely.
I will review how to support 2016 device.

Thank you.

**Comment by vitalets** [comment]
@pwsses could you provide some more information why 2016 is not supported? It seems there are some differences for `sdb` syntax, but all required operations (push, debug, connect ws) should be possible..

**Comment by jfrux** [comment]
I’m thoroughly interested in this as the current deployment strategy / dev strategy using Tizen Studio is atrocious for web developers and single code bases.
I find it to be too inconsistent.

I may have to install like Tizen Studio 1.3 or something to get our app testing going on 2016 TV.

--
Joshua F. Rountree
UX Developer
Newsy, The E.W. Scripps Company
312 Walnut Street, Suite 2800, Floor 29th, Office 29W109
Cincinnati, Ohio 45202-4024
(513) 977-3840 (office, M-F, 7a-4p)
(513) 827-7936 (mobile)


Newsy is America’s fastest-growing news network

**Comment by jfrux** [comment]
@vitalets I'd love to know this info as well...
I'm now having trouble getting apps to install at all on 2016 TV from Tizen Studio 5 updates...

**Comment by pwsses** [comment]
@jfrux You mean after Tizen Studio 5 updates, failed install at app on 2016 TV? Without regard to Wits?

**Comment by jfrux** [comment]
@pwsses I dunno what I mean anymore... I think I'm beginning to lose it...
I'm surrounded by TVs.... my world is spinning..... out of control...... tooo mannny TVs.....
Ahhhhh!

All I see when I sleep is TVs....
💤📺

I think they just kind of forgot to test 2016's...
Once I got my 2018 in the office I was able to install with Wits and my own scripts...
Thank God.

I'm sorry for blowing up your repo... was kind of getting desperate.
